lib/language_pack/installers/heroku_ruby_installer.rb

```diff
@@ -1,25 +1,55 @@
-require 'language_pack/installers/ruby_installer'
 require 'language_pack/base'
 require 'language_pack/shell_helpers'
 
-class LanguagePack::Installers::HerokuRubyInstaller
-  include LanguagePack::ShellHelpers, LanguagePack::Installers::RubyInstaller
+module LanguagePack::Installers; end
 
+class LanguagePack::Installers::HerokuRubyInstaller
   BASE_URL = LanguagePack::Base::VENDOR_URL
+  BIN_DIR = Pathname("bin")
+
+  include LanguagePack::ShellHelpers
+  attr_reader :fetcher
 
-  def initialize(stack)
-    @fetcher = LanguagePack::Fetcher.new(BASE_URL, stack)
+  def initialize(stack: )
+      @fetcher = LanguagePack::Fetcher.new(BASE_URL, stack: stack)
```

Comment:
This looks over-indented?

```suggestion
    @fetcher = LanguagePack::Fetcher.new(BASE_URL, stack: stack)
```